airflow/providers/presto/hooks/presto.py

```diff
@@ -111,27 +112,87 @@ def get_isolation_level(self) -> Any:
     def _strip_sql(sql: str) -> str:
         return sql.strip().rstrip(';')
 
-    def get_records(self, hql, parameters: Optional[dict] = None):
-        """Get a set of records from Presto"""
+    @overload
+    def get_records(self, sql: str = "", parameters: Optional[dict] = None):
+        """Get a set of records from Presto
+
+        :param sql: SQL statement to be executed.
+        :param parameters: The parameters to render the SQL query with.
+        """
+
+    @overload
+    def get_records(self, sql: str = "", parameters: Optional[dict] = None, hql: str = ""):
+        ...
+
+    def get_records(self, sql: str = "", parameters: Optional[dict] = None, hql=None):
```

Comment:
```suggestion
    def get_records(self, sql: str = "", parameters: Optional[dict] = None, hql: str = ""):
```

This does not need to change?

Reply:
I can change that but the second overload and the actual implementation will have the same signature, it can be confusing?

Reply:
Also, I think the idea here is to have the most loose signature in actual implementation like args and kwargs or union of them. 
So, below should be the code of actual implementation :
`def get_records(self, sql: str = "", parameters: Optional[dict] = None, **kargs)`

Reply:
The two are used for different purposes; the overload is only used by Mypy, and the actual implementation is only used at runtime. So having them match is not a problem.

Reply:
Sure, I'll make update the PR.